packages/likec4/src/vite-plugin/virtuals/icons.ts

```diff
@@ -8,18 +8,33 @@ function code(views: ComputedView[]) {
   const icons = pipe(
     views.flatMap(v => v.nodes.map(n => n.icon)),
     filter(isString),
-    filter(s => isTruthy(s) && !s.toLowerCase().startsWith('http')),
+    filter(s =>
+      isTruthy(s) &&
+      !(s.toLowerCase().startsWith('http:') || s.toLowerCase().startsWith('https:'))
+    ),
     unique(),
   ).sort()
 
   const {
     imports,
     cases,
   } = icons.reduce((acc, s, i) => {
-    const [group, icon] = s.split(':') as ['aws' | 'azure' | 'gcp' | 'tech', string]
-
+    const isLocalImage = s.startsWith('file:')
     const Component = 'Icon' + i.toString().padStart(2, '0')
 
+    if (isLocalImage) {
+      // Here the path can't be found, presumably because the file
+      // is not in the same directory as the code that's running.
+      // We need to find the path of the original file that contained
+      // the model, and then make the path (s) relative to that.
+      acc.imports.push(`import ${Component} from '${s}?inline'`)
+      acc.cases.push(`  '[${s}]': () => <img src={${Component}} />`)
```

Comment:
```suggestion
      acc.cases.push(`  '${s}': () => jsx('img', { src: ${Component}})`)
```

`[${s}]` is invalid.

Could you try this? (Also, seems we can't use jsx syntax)